### DISCUSSION TITLE
[cloud_firestore] add `in`, `not-in`, and `array-contains-any` operators

### DISCUSSION THREAD

**Discussion by ming-chu** [firebase/flutterfire#3987] [Feature request] [answered]
According to the [official docs](https://firebase.google.com/docs/firestore/query-data/queries#in_and_array-contains-any), `Cloud Firestore` support `in`, `not-in`, and `array-contains-any` operators now. 

[Cloud Firestore now supports not equal queries](https://firebase.googleblog.com/2020/09/cloud-firestore-not-equal-queries.html)

Let's consider supporting those new operators for `cloud_firestore` plugin?

**Comment by Salakar** [comment, marked as answer]
This is now supported in the latest release. Thanks
